### PR TITLE
Balsamic UMI & QC pipelines

### PR DESCRIPTION
**Pull request by ivadym**
### Added
- Balsamic UMI and QC pipeline specific implementations

---

### How to prepare for test
- [x] ssh to hasta.scilifelab.se
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-hermes-stage.sh [THIS-BRANCH-NAME]`

### Tests

- Test deliverables for `TN-PANEL` and `balsamic`, `balsamic-umi` & `balsamic-qc` pipelines:

Screen Shot 2022-05-31 at 09 25 31

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Comment by ivadym** [comment]
Test deliverables for `TN-PANEL` and `balsamic` & `balsamic-umi` pipelines:

Screen Shot 2022-05-20 at 16 50 46

File: cg_hermes/config/balsamic_qc.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+from cg_hermes.config.balsamic import QC_TAGS, RAW_TAGS
+
+
+QC_ALIGNMENT_TAGS = {
+    # Alignment files (PANEL & WGS)
+    frozenset(RAW_TAGS["tumor.merged.cram"]): {  # cram (tumor)
+        "tags": ["tumor", "qc-cram"],
+        "is_mandatory": True,
+        "used_by": ["deliver", "scout"],
```
Comment (ashwini06) [inline cg_hermes/config/balsamic_qc.py:15]: For balsamic-qc analysis, does it makes sense to upload only cram files to Scout? We don't have VCFs for this analysis. may be only caesar delivery isn't good enough??

File: docs/balsamic_map.md [hunk 2]
```diff
@@ -1,43 +1,49 @@
...
-| diagram, cnv-diagram                                           | False       | cnvkit, visualization, diagram      | deliver               |
-| cnv-gene-metrics, gene-metrics                                 | False       | cnvkit, metrics, genes              | deliver               |
-| cgh-file, cnv-somatic-cgh-file                                 | False       | cnvkit, vcf2cytosure                | deliver, scout        |
-| clinical-rd-delly, rd-delly                                    | False       | delly, metrics                      | deliver               |
+| Balsamic tags                                                  | Mandatory   | HK tags                                 | Used by               |
+|----------------------------------------------------------------|-------------|-----------------------------------------|-----------------------|
+| balsamic-config                                                | True        | balsamic-config                         | audit, cg             |
+| balsamic-report                                                | True        | balsamic-report                         | audit                 |
```
Comment (ashwini06) [inline docs/balsamic_map.md:4]: Is this the delivery report? If so, will it be also uploaded to scout and deliver?

**Review by ashwini06 — APPROVED** [review]
Looks good 💯  Just few minor comments which are not clear to me.

File: docs/balsamic_map.md (hunk 2 above)
Comment (ivadym) [reply docs/balsamic_map.md:4]: No, the delivery report generation, storage and delivery logics are in CG. This is just the snakemake html report located in `/analysis/delivery_report/` that we are creating with (`balsamic report deliver ...`).

**Comment by ivadym** [comment]
Screen Shot 2022-07-05 at 13 28 43

Screen Shot 2022-07-05 at 13 37 13

**Pull request merged**